### PR TITLE
Adapt examples to new app manifest concept

### PR DESCRIPTION
**Pull request by BjoernAtBosch**
## Describe your changes

Adapt AppManifests of examples to new app manifest concept

## Issue ticket number and link

AB#12908

## Checklist - Manual tasks

* [x] Examples are executing successfully
* [ ] Created/updated unit tests. Code Coverage percentage on new code shall be >= 80%.
* [ ] Created/updated integration tests.
* [x] Devcontainer can be opened successfully
* [x] Devcontainer can be opened successfully behind a corporate proxy
* [x] Devcontainer can be re-built successfully
* [ ] Extended the documentation (e.g. README.md, CONTRIBUTING.md, Velocitas)

### CODE REVIEW

File: examples/array-datatype/AppManifest.json [hunk 1]
```diff
@@ -1,40 +1,18 @@
 [
     {
         "Name": "array-datatype",
-        "Port": 50008,
-        "DAPR_GRPC_PORT": 50001,
```
Comment (MP91) [inline examples/array-datatype/AppManifest.json:5]: Is it already clarified where we put these values instead?

Comment (MP91) [reply examples/array-datatype/AppManifest.json:5]: As discussed, this is hardcoded for now in the devenv-runtime scripts and will be solved later with the middleware-selection

**Review by MP91 — APPROVED** [review]
I guess we should wait with merging this until we have finalized the other PRs

**Review by kse3hi — APPROVED** [review]
LGTM

**Pull request merged**